manifests/server/kickstart/runpuppet.pp

```diff
@@ -0,0 +1,86 @@
+# This class manages the runpuppet script, which is a script that can be run
+# to bootstrap provisioned systems, adding them to puppet and running it in a
```

Comment:
We need a style guide for this sort of thing. I know in SIMP we tend to prefer clients, but other people would call them nodes, systems, boxes, or any number of things. I will change to clients for now.